### DISCUSSION TITLE
Do jobrunr supports recurring jobs with start / end date-time?

### DISCUSSION THREAD

**Discussion by rkuzmych** [jobrunr/jobrunr#281] [Q&A] [answered]
JobRunr Version: 4.0.0

As i undrerstand, it is not supported yet? Something like this:

`scheduler.scheduleRecurrently(jobId, cronExpression, timeZone, JobRequest, startTime, endTime.);`
There are no ways to configure start time of created job by filters?

**Comment by rdehuyss** [comment]
I don't understand your question... you have a cron expression so why would you need startTime and endTime?

**Comment by rdehuyss** [comment, marked as answer]
As it is now and the feature you're looking for is not there, it will probably also not be added immediately.

**Reply by rkuzmych** [reply]
Thank you for answer, i appreciate it!
